src/SurveyService.Validation/UserAnswer/CreateUserAnswerRequestValidator.cs

```diff
@@ -0,0 +1,45 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.Kernel.Extensions;
+using LT.DigitalOffice.SurveyService.Data.Interfaces;
+using LT.DigitalOffice.SurveyService.Models.Dto.Requests.UserAnswer;
+using LT.DigitalOffice.SurveyService.Validation.UserAnswer.Interfaces;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace LT.DigitalOffice.SurveyService.Validation.UserAnswer;
+
+public class CreateUserAnswerRequestValidator : AbstractValidator<CreateUserAnswerRequest>, ICreateUserAnswerRequestValidator
+{
+  public CreateUserAnswerRequestValidator(
+    IOptionRepository _optionRepository,
+    IHttpContextAccessor _httpContextAccessor)
+  {
+    RuleFor(request => new
+    {
+      optionsIds = request.OptionIds,
+      options = _optionRepository.GetByIdsAsync(request.OptionIds).Result
+    })
+      .Cascade(CascadeMode.Stop)
+      .Must(x => x.optionsIds.Any())
+      .WithMessage("There are no optionIds.")
```

Comment:
idk, but maybe it's better to write something more understandable for developers, who have no idea about what is optionIds?

Reply:
The same applies to other messages

Reply:
Done